public/javascripts/ng-apps.js

```diff
@@ -1,17 +1,21 @@
-var springInitiative = angular.module('springInitiative', ['ngRoute']);
+var springInitiative = angular.module('springInitiativeApp', ['ui.router']);
 
-springInitiative.config(function($routeProvider) {
-  $routeProvider
-    .when('/student', {
-      templateUrl: 'views/student.html',
+springInitiative.config(function($stateProvider, $urlRouterProvider) {
+
+  $urlRouterProvider.otherwise('/login');
+
+  $stateProvider
+
+    .state('student', {
+      url: '/student',
+      templateUrl: 'views/partial-student.html',
```

Comment:
since everything will be partial do we want to have partial in their names? should we have it as `views/student.html` or `views/partial/student.html` or this